hadoop-ozone/dist/src/main/smoketest/recon/recon-api.robot

```diff
@@ -34,6 +34,13 @@ Recon REST API
                         Should contain      ${result}       ozone_datanode_1.ozone_default
                         Should contain      ${result}       ozone_datanode_2.ozone_default
                         Should contain      ${result}       ozone_datanode_3.ozone_default
+    ${result} =         Execute                             curl --negotiate -u : -v ${API_ENDPOINT_URL}/pipelines
+                        Should contain      ${result}       pipelines
+                        Should contain      ${result}       RATIS
+                        Should contain      ${result}       OPEN
```

Comment:
Added a 2s delay before executing CURL calls. I don't think this will be flaky since recon tests will run after several other tests in both `ozone` and `ozone-secure` docker-compose. That should give recon a lot of time to get heartbeats. I have also run this test locally with recon tests running first after docker setup is up and haven't faced any flakiness issues. I am positive that this will not be flaky.

Reply:
Makes sense. I am ok with this.